mavlink/definitions/uavionix.xml

```diff
@@ -0,0 +1,399 @@
+<?xml version='1.0'?>
+<mavlink>
+    <include>common.xml</include>
+    <enums>
+      <enum name="UAVIONIX_SDA_LEVEL">
```

Comment:
I agree, it's unfortunate that there aren't any real standards in any of these definition files (or MAVLink as a whole).